### PR TITLE
Multiple updates to pyGDP

### LINKED ISSUE #142: Update job monitoring to be more robust.
@pnorton-usgs has implemented some functionality to return an object that represents a running job. Similar to how geoknife works. It would be great to get this functionality into pyGDP to replace how the existing job monitoring works in _executeRequest.

### LINKED ISSUE #147: Rename CSWURL to CSW_URL in namespaces.py
Rename CSWURL to CSW_URL in namespaces.py which is the general format used by the other *_URL variables.

### LINKED ISSUE #148: _executeRequest() does not pass verbose setting to the WebProcessingService()
The _executeRequest() method in _executeRequest.py does not pass the verbose parameter setting on to the WebProcessingService().

### LINKED ISSUE #149: Inconsistent used of coverage=True in various methods
Sometimes where a method uses the coverage parameter it defaults to True and other times it defaults to 'true'. There is inconsistencies in how code that relies on this parameter variable handle it.

### PR DESCRIPTION
**Pull request by pnorton-usgs**
Lot's of PEP8-related changes to variables within methods #146.

Added async=False to _executeRequest() method (fixes #142). The default behavior is to block and return a file when processing completes. By setting async=True a wps.execute object is returned which can be monitored in a non-blocking manner. This allows greater control and flexibility for long-running GDP jobs while maintaining the original behavior.

Renamed 'pygdp' directory to 'pyGDP'. Moved pyGDP.py and GDP_XML_Generator.py into pyGDP directory. 

Renamed pyGDP.py to pyGDPwebProcessing.py. Pulled namespace-related imports out of pyGDPwebProcessing.py and put in pyGDP/__init__.py

Updated imports in all files to reflect structure changes. Removed or commented out unnecessary imports.

Renamed CSWURL to CSW_URL in namespaces.py (fixes #147).

Modified _executeRequest() to pass verbose parameter on to the WebProcessingService() (fixes #148).

In feature_coverage.py changed parameter variables coverage='true' to coverage=True for methods submitFeatureCoverageOPenDAP(), submitFeatureCoverageWCSIntersection(), submitFeatureCategoricalGridCoverage(). Modified code that works with the coverage variable (fixes #149).

### CODE REVIEW

**Comment by dblodgett-usgs** [comment]
@ocefpaf - would you be willing to have a look at this PR? We don't have a lot of spare cycles for code review and you are probably the closest to this in recent time. Thanks!

**Comment by ocefpaf** [comment]
TL;DR 

If we trust the test coverage let's merge this. I don't think anyone has cycles to go over the coding standards changes.

---
Indeed +1,502 −1,245 change sets, 7 commits, and 54 files changed is not an easy review!

Ignoring the first two commits (`code formatting`), merge remote-tracker, and the two `lots of changes` (:smile:) we have:

- Fixing the typo `WPSgetCapabilities` changes the API; I am :+1: to the change but that needs to be documented somewhere.

@pnorton-usgs the scenario(s) that are not working are due code issues/bugs or server problems?
Not sure how lettuce works with the `@not_working` decorator, but if it is like a py.test mark maybe we could use a better name like `@server_failure`?

Everything else seems fine and I guess it was an Herculean job to bring the code formatting up to standards in one go! Thanks @pnorton-usgs!!

Note that even though I am always :+1: for doing that, the changes must be smooth if one is to review them in a PR.

**Comment by pnorton-usgs** [comment]
@ocefpaf the scenario that is not working was due to a single server. I don't know lettuce very well but I did not see a way to set a single server as not_working. All other servers for this scenario worked just fine but I ended up setting the whole scenario as not working.

I'll make sure to keep my future commits/PRs to a reasonable size in the future; this one is painful even for me. :)
